venia-integration-tests/src/fields/wishlist/index.js

```diff
@@ -7,10 +7,15 @@ export const wishlistItemPrice = '*[class^="wishlistItem-priceContainer-"]';
 export const wishlistItemAddToCartButton =
     '*[class^="wishlistItem-addToCart-"]';
 
+export const wishlistItemRemoveButton = '*[class^="wishlistItem-delteItem-"]';
```

Comment:
This selector is accurate, but maybe we should fix the typo in the components className.

```suggestion
export const wishlistItemRemoveButton = '*[class^="wishlistItem-deleteItem-"]';
```

Reply:
You are right, i have fixed it in 46bc993